src/hidimstat/permutation_importance.py

```diff
@@ -53,6 +53,7 @@ def __init__(
             n_permutations=n_permutations,
         )
         self.rng = check_random_state(random_state)
```

Comment:
If you register, `random_state`, you should remove `self.rgn` if it's not necessary, in other cases, you can replace it with `self._rng`

Reply:
done